Source/WebKit/GPUProcess/mac/com.apple.WebKit.GPUProcess.sb.in

```diff
@@ -84,17 +84,16 @@
         "AppleM2ScalerCSCDriver"
         "IOPMrootDomain"))
 
+
+;; FIXME: Add check `(with-filter (system-attribute virtual-device)` when this is supported. See rdar://118274636.
+(allow iokit-open-user-client (iokit-user-client-class “AppleVideoToolboxParavirtualizationUserClient"))
```

Comment:
It would be nice to move the existing `AppleVideoToolboxParavirtualizationDriver` entry next to this, so that we remember to conditionalize both in the future. Also, there is some potential cleanup in WebContent process sandbox.

Reply:
That is a good point. I will fix this before landing!